adagrams/game.py

```diff
@@ -1,11 +1,146 @@
+import random 
+import string
```

Comment:
[nit] Unneeded import. This probably got added when you named a variable `string`, then renamed it `strings`.